Fixed grammatical errors and typos in: CONTRIBUTING.md README.md inde…

AlligatorSnappingTurtle19 opened the pull request:
I fixed grammatical errors and typos that I was able to find in the files titled: 'CONTRIBUTING.md', 'README.md', and 'index.html'.

AlligatorSnappingTurtle19 commented on index.html:
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```
I am not sure why GitHub and GitBash are both insisting that there was a change made on this line. However, I do not see any difference between the two lines so I doubt it can cause a problem, or modify the application's actions.

amitguptagwl commented on CONTRIBUTING.md:
```diff
@@ -1,14 +1,14 @@
+This application is made with the help of HTML, CSS, vanilla JS, Jquery, and Riot js.
 
-This guide will may help first timers about what to do other than code change.
+This guide may help first timers about what to do other than change the code.
```
I believe the correct sentence would be "This guide may help first timers about what to do other than code change" or "This guide may help first timers about what to do other in addition of code change"

AlligatorSnappingTurtle19 commented:
I created a new version that adds the line 'This guide may help first timers about what to do other than code change', added the proper name of the application on line 6 of the README.md, and I also switched all the 'symbol's to 'image'. I changed them because I felt that within the context of the application using 'image' over 'symbol' made more sense. However, if you would rather me use 'symbol' instead of 'image' please alert me and I will make the necessary changes.

amitguptagwl commented:
Thanks @AlligatorSnappingTurtle19 for your detailed review of the documentation. 'image' sounds non technical term. And this is what we want.
